scripts/download_datasets.py

```diff
@@ -0,0 +1,34 @@
+"""
+Helper script to download OmniGibson dataset and assets.
+"""
+import os
+os.environ["OMNIGIBSON_NO_OMNIVERSE"] = "1"
+
+from omnigibson.macros import gm
+from omnigibson.utils.asset_utils import download_og_dataset, download_assets
+import click
+
+
+def main():
+    # Ask user which dataset to install
+    data_path = gm.DATA_PATH if os.path.isabs(gm.DATA_PATH) else os.path.abspath(f"omnigibson/{gm.DATA_PATH}")
```

Comment:
IIRC these data paths should already be modified at runtime so that they're always an absolute path

Reply:
reverted change